docs/content/docs/templates.md

```diff
@@ -482,6 +482,43 @@ Returns the length of an array or a string, 0 if the value is not an array.
 ### reverse
 Returns a reversed string or array.
 
+### sort
+Sorts an array into ascending order.
+
+The values in the array must be a sortable type:
+- numbers are sorted by their numerical value.
+- strings are sorted in alphabetical order.
+- arrays are sorted by their length.
+- bools are sorted as if false=0 and true=1
```

Comment:
now that i think of it, does it ever makes sense to sort an array of bool?

Reply:
It's probably a pretty rare use case. But I don't think it would hurt to have the ability to do it.